src/ActionList/Item.tsx

```diff
@@ -457,4 +478,6 @@ export function Item(itemProps: Partial<ItemProps> & {item?: ItemInput}): JSX.El
       </DividedContent>
     </StyledItem>
   )
-}
+}) as PolymorphicForwardRefComponent<'div', ItemProps>
+
+export {Item}
```

Comment:
Is there a reason you wrote the export like this instead of doing: `export const Item ...` on line 318

Reply:
Nope, will fix.